person.rb

```diff
@@ -0,0 +1,26 @@
+class Person
+  attr_reader :id
+  attr_accessor :name, :age
+
+  def initialize(age, name = 'Unknown', parent_permission: true)
+    @id = rand(1...1000)
+    @name = name
+    @age = age
+    @parent_permission = parent_permission
+  end
+
+  def can_use_services?
+    isof_age? || "#{@parent_permission}, He has parental permission"
```

Comment:
- [ ] This method should return a `boolean` not a string kindly change this and leave `@parent_permission` only. 🤓

Reply:
done :+1: